manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "b2b-quotes-graphql",
-  "version": "2.2.4",
+  "version": "2.3.0",
```

Comment:
Can you revert these version version bumps from here and the package.json? They are done automatically by vtex io bot on merging to master, we shouldn't change these ourselves.

Reply:
It did it when I merged another branch in this. I'll revert it